babel_monitor/src/lib.rs

```diff
@@ -186,12 +189,13 @@ impl<T: Read + Write> Babel<T> {
             if entry.contains("add neighbour") {
                 vector.push_back(Neighbor {
                     id: find_babel_val("neighbour", entry)?,
+                    address: find_babel_val("address", entry)?.parse()?,
                     iface: find_babel_val("if", entry)?,
                     reach: u16::from_str_radix(&find_babel_val("reach", entry)?, 16)?,
                     txcost: find_babel_val("txcost", entry)?.parse()?,
                     rxcost: find_babel_val("rxcost", entry)?.parse()?,
-                    rtt: find_babel_val("rtt", entry)?.parse()?,
-                    rttcost: find_babel_val("rttcost", entry)?.parse()?,
+                    rtt: 0.0,
```

Comment:
@drozdziak1  fixing these in the firmware and the like is part of the task of full path rtt.

Reply:
@drozdziak1 is #97 ready for me to merge/base on? I'm really trying to cut a stable version this week so I don't want to merge bigger changes.